engines/wintermute/wintermute.cpp

```diff
@@ -158,6 +162,17 @@ int WintermuteEngine::init() {
 		}
 	#endif
 
+	// check dependencies for games with FoxTail subengine
+	#if not defined(ENABLE_FOXTAIL)
+		if (BaseEngine::isFoxTailCheck(_gameDescription->targetExecutable)) {
+			GUI::MessageDialog dialog(_("This game requires FoxTail subengine, that is not compiled in."));
```

Comment:
“This game requires the Foxtail subengine...”

Reply:
also, "which is not"?

Reply:
Thanks! 

Typos and bad english fixed at 8ab48532597a04ef749b2e16b8417f4158a08039.